[FIX] subscriptions: updated and corrected mistake re: eLearning (#8293)
odoo/documentation: content/applications/sales/subscriptions/ecommerce.rst

Create recurrence periods as product variants
=============================================

To set up each recurrence period as a product variant, go to :menuselection:`Subscriptions app -->
Products`, and select a product. In the :guilabel:`Attributes & Variants` tab, click :guilabel:`Add
a line`.

> Then proceed to create an :guilabel:`Attribute` called `Billing Period` (or something similar), by
> typing in the name, and clicking :guilabel:`Create` from the mini drop-down menu that appears. This
> attribute name appears as option heading on the product page of the eCommerce shop.

Reviewer: Totally personal preference, but when it comes to naming things, since users don't HAVE to call it `Billing Period`, let's try describing what they should type in, instead of telling them what to type.
```suggestion
Then proceed to create an :guilabel:`Attribute` by typing in a name to represent the billing period, and clicking :guilabel:`Create` from the mini drop-down menu that appears. This
attribute name appears as option heading on the product page of the eCommerce shop.
```

Author: I totally understand (and appreciate) where you're coming from. I, too, feel that same way. Personally, I think my parenthetical suggestion of "(or something similar)" suffices in this regard, and doesn't make the user feel beholden to one *specific* title. So, I think I'm going to leave it as-is, but thanks for the suggestion. 👍

the :guilabel:`Recurring Prices` tab of the product form.

To do that, type in the name of the recurrence period in the :guilabel:`Values` column, on the same
:guilabel:`Attribute` line, in the :guilabel:`Attributes & Variants` tab. Then, click
:guilabel:`Create` from the mini drop-down menu that appears.

These value names appear as selectable options on the product page of the eCommerce shop.

.. image:: ecommerce/recurrence-period-attributes-variants.png
   :align: center
   :alt: Recurrence periods configured as product variants in the "Attributes & Variants" tab of
         the product form.

With those configurations in place and saved, a :guilabel:`Product Variants` column appears in the
:guilabel:`Recurring Prices` tab. Proceed to assign the different :guilabel:`Product Variants` to
their corresponding recurrence :guilabel:`Period` and :guilabel:`Price`.

.. image:: ecommerce/product-variants-time-based-pricing.png
   :align: center
   :alt: Product variants on the "Time-based pricing" tab of the product form.

After following those aforementioned steps, the product variants are available for selection on the

> eCommerce product page.

Reviewer: it might be helpful to include menuselection steps to get to the eCommerce product page?

Author: This would be a difficult task, as they can implement the lesson from this doc on *any* subscription product, which could be on *any* page, of their online/eCommerce store. So, I think this more generic allusion to where it would be located is the best/most apt approach for this particular circumstance.
